doc/classes/ProjectSettings.xml

```diff
@@ -875,6 +875,10 @@
 			[b]Note:[/b] Certain window managers can be configured to ignore the non-resizable status of a window. Do not rely on this setting as a guarantee that the window will [i]never[/i] be resizable.
 			[b]Note:[/b] This setting is ignored on iOS.
 		</member>
+		<member name="display/window/size/sharp_corners" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], the main window uses sharp corners by default.
+			[b]Note:[/b] This property is implemented only on Windows.
```

Comment:
```suggestion
			[b]Note:[/b] This property is implemented only on Windows, and only has an effect on Windows 11. Previous Windows versions always use sharp corners regardless of this setting.
```

Reply:
I was wondering whether to point this out, but dismissed my own comment because "sharp corners" is exactly how it is for any prior version, so the addendum feels redundant to me.

Regardless this suggestion is a bit too verbose.
```suggestion
			[b]Note:[/b] This property is implemented only on Windows 11 or newer. Previous Windows versions always use sharp corners regardless of this setting.
```
But again, eh, not sure.

Reply:
I suppose it could be consolidated to something like what other OSes do in documentation: specifying the relevant edition/version in parenthesis.
```diff
-			[b]Note:[/b] This property is implemented only on Windows, and only has an effect on Windows 11. Previous Windows versions always use sharp corners regardless of this setting.
+			[b]Note:[/b] This property is implemented only on Windows (11).
```